docs/querying/aggregations.md

```diff
@@ -422,6 +389,121 @@ It is not possible to determine a priori how well this aggregator will behave fo
 
 For these reasons, we have deprecated this aggregator and recommend using the DataSketches Quantiles aggregator instead for new and existing use cases, although we will continue to support Approximate Histogram for backwards compatibility.
 
+
+## Expression aggregators
+
+### Expression aggregator
```

Comment:
```suggestion
```
I think the heading on line 393 is sufficient.

Reply:
i considered both this aggregator and the javascript aggregator as free form "expression" aggregators since you can just write whatever functions you want, which is why they are both under the category. can you think of a better category name than "Expression aggregators"? The native expression aggregator and javascript aggregator are totally separate, just similar in spirit...